pyrate/core/gamma.py

```diff
@@ -188,14 +284,108 @@ def combine_headers(hdr0, hdr1, dem_hdr):
         args = (chdr[ifc.FIRST_DATE], chdr[ifc.SECOND_DATE])
         msg = "Wavelength mismatch, check both header files for %s & %s"
         raise GammaException(msg % args)
+
+    # use parameter of first image (as done by GAMMA during interferometric processing)
+    heading_ang = hdr0[ifc.PYRATE_HEADING_DEGREES]
+    if np.isclose(heading_ang, hdr1[ifc.PYRATE_HEADING_DEGREES], atol=1e-1):
+        chdr[ifc.PYRATE_HEADING_DEGREES] = heading_ang
+    else:
+        args = (chdr[ifc.FIRST_DATE], chdr[ifc.SECOND_DATE])
+        msg = "Satellite heading angles differ by more than 1e-1"
```

Comment:
I wonder whether tiis worth making these error messages a bit more "human readable", for instance:

```
msg = "Satellite heading angles differ by more than 1 degree"

msg = "SAR to Earth Center distance differs by more than 1000 metres"
```

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee